src/core.js

```diff
@@ -482,14 +482,17 @@ $.extend( $.validator, {
 			this.submitted = {};
 			this.prepareForm();
 			this.hideErrors();
-			var i, elements = this.elements()
+			var elements = this.elements()
 				.removeData( "previousValue" )
 				.removeAttr( "aria-invalid" );
 
+			this.resetElements( elements );
+		},
+
+		resetElements: function( elements ) {
 			if ( this.settings.unhighlight ) {
-				for ( i = 0; elements[ i ]; i++ ) {
-					this.settings.unhighlight.call( this, elements[ i ],
-						this.settings.errorClass, "" );
+				for ( var i = 0; elements[ i ]; i++ ) {
```

Comment:
Yes, there is a JSCS rule that disallow that: [requireVarDeclFirst](http://jscs.info/rule/requireVarDeclFirst)

But using it will require fixing a lot of issues in test and src files.

What do you think ?